src/data.py

```diff
@@ -261,6 +268,22 @@ def parse_static_op_hours(hours_list, eatery_id, dining_items):
     )
   return new_operating_hours
 
+def is_closed(current_date, dates_closed):
+  """Returns boolean indicating if the specified date is present in a list of closed dates.
+  Parameter current_date: a datetime.date object of the date to compare
+  Parameter dates_closed: a list of strings either in form 'mm/dd/yy' for a singular date or
+  'mm/dd/yy-mm/dd/yy' for a range of dates.
+  """
+  for dates in dates_closed:
+    if '-' not in dates and current_date.strftime('%m/%d/%y') == dates:
+      return True
+    start_date, end_date = dates.split('-')
```

Comment:
generally i think it's best not to reassign a var like this, especially since it's changing types (from string to Date). python's lack of typed variables is probably one of the worst things about it and we should generally stay away from relying on it